src/hooks/useActiveIfoWithBlocks.ts

```diff
@@ -31,5 +32,5 @@ export const useActiveIfoWithBlocks = (): Ifo & { startBlock: number; endBlock:
     },
   )
 
-  return activeIfo ? { ...activeIfo, ...currentIfoBlocks } : null
+  return activeIfo ? { ...activeIfo, ...currentIfoBlocks, isLoading: status !== FetchStatus.Fetched } : null
```

Comment:
Could you help me add memo for this ?

Thanks

Reply:
@chefilip I am not sure what your mean, you mean memo the return object?

Reply:
@chefilip I am not sure what your mean, you mean memo the return object?

Reply:
Yeah. I mean memo the return object.